recipes/flake8-isort/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "flake8-isort" %}
+{% set version = "2.7.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 81a8495eefed3f2f63f26cd2d766c7b1191e923a15b9106e6233724056572c68
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python
+    - setuptools
+  run:
+    - flake8 >=3.2.1
+    - isort >=4.3.0
+    - python
+    - setuptools
+    - testfixtures
```

Comment:
It does need this module to run this project despite the name that it seems to be just for testing.